CMakeLists.txt

```diff
@@ -155,6 +155,17 @@ GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL for details.
 ]==])
 mark_as_advanced(GOOGLE_CLOUD_CPP_OVERRIDE_GOOGLEAPIS_URL_HASH)
 
+option(
+    GOOGLE_CLOUD_CPP_USE_INSTALLED_COMMON
+    [==[
+Use a pre-installed version of the common libraries and proto files.
+
+This option is required to support package maintainers that split
+`google-cloud-cpp` into multiple (hopefully smaller) packages.
```

Comment:
Is it even possible to split into larger packages?

Reply:
Fixed.